src/serializers/persons-serializer.js

```diff
@@ -0,0 +1,83 @@
+import { Serializer as JsonApiSerializer } from 'jsonapi-serializer';
+import _ from 'lodash';
+import merge from 'merge-deep';
+import moment from 'moment';
+
+import { serializerOptions } from 'utils/jsonapi';
+import { openapi } from 'utils/load-openapi';
+import { apiBaseUrl, resourcePathLink } from 'utils/uri-builder';
+
+const personResourceProp = openapi.components.schemas.PersonResult.properties.data.properties;
+const personResourceType = personResourceProp.type.enum[0];
+const personResourceAttributes = personResourceProp.attributes.allOf;
+const personCombinedAttributes = merge(personResourceAttributes[0], personResourceAttributes[1]);
```

Comment:
Not sure, but can't we use [lodash.merge](https://lodash.com/docs/#merge)?

Reply:
I swear I looked at lodash for this. Yeah it works.